src/pages/Mission.js

```diff
@@ -0,0 +1,52 @@
+import { React, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { FetchMissionsHandler, ToggleMission } from '../redux/missions/mission';
+import './mission.css';
+
+const Mission = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missionsReducer);
+  useEffect(() => {
+    if (!missions.length) dispatch(FetchMissionsHandler());
+  }, []);
+  const HandleClick = (id) => dispatch(ToggleMission(id));
+
+  return (
+    <div className="missions">
+      <table>
+        <thead className="tableHeader">
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {missions.map((el) => (
+            <tr key={el.mission_id}>
+              <td className="missionName">{el.mission_name}</td>
+              <td className="missionDescription">{el.description}</td>
+              <td className="btnRow">
+                <span className={el.active ? 'activeMember' : 'inactiveMember'}>
+                  {el.active ? 'Active Member' : 'Not A member'}
```

Comment:
- [ ] Please fix the issue with the text overlapping for some sizes of screens. Will create a better UI.

<img width="498" alt="Screen Shot 2022-11-19 at 2 46 40 PM" src="https://user-images.githubusercontent.com/47673311/202864562-0a8b8f3f-573b-4dcd-aa9b-bc3fa088539b.png">

Reply:
The project doesn't require us to have responsiveness